### DISCUSSION TITLE
Jedis version to support Redis 7.1.0, AWS ElastiCache

### DISCUSSION THREAD

**Discussion by davis0119** [xetorthio/jedis#3802] [Q&A]
I am having issues connecting to q AWS ElastiCache Redis cache with error

```
"message": "java.net.SocketTimeoutException: Read timed out",
"name": "redis.clients.jedis.exceptions.JedisConnectionException"
```

I am using JedisPool with dependency `"redis.clients" % "jedis" % "5.1.0"`. I have tried other versions but noticed on the readme that "The most recent version of this library supports redis version [5.0](https://github.com/redis/redis/blob/5.0/00-RELEASENOTES), [6.0](https://github.com/redis/redis/blob/6.0/00-RELEASENOTES), [6.2](https://github.com/redis/redis/blob/6.2/00-RELEASENOTES), [7.0](https://github.com/redis/redis/blob/7.0/00-RELEASENOTES) and [7.2](https://github.com/redis/redis/blob/7.2/00-RELEASENOTES)."

Is version 7.1 not supported?

**Comment by sazzad16** [comment]
There is no Redis 7.1.
